Formula/cataclysm.rb

```diff
@@ -1,25 +1,23 @@
 class Cataclysm < Formula
   desc "Fork/variant of Cataclysm Roguelike"
   homepage "https://github.com/CleverRaven/Cataclysm-DDA"
-  url "https://github.com/CleverRaven/Cataclysm-DDA/archive/0.D.tar.gz"
-  version "0.D"
-  sha256 "6cc97b3e1e466b8585e8433a6d6010931e9a073f6ec060113161b38052d82882"
-  revision 1
+  url "https://github.com/CleverRaven/Cataclysm-DDA/archive/0.E.tar.gz"
+  version "0.E"
+  sha256 "b0af9a9292929e17332edcea770bca9a91f1d08ea47726d78a47e09281a42fa3"
   head "https://github.com/CleverRaven/Cataclysm-DDA.git"
 
   bottle do
     cellar :any
     rebuild 1
```

Comment:
This can be removed.

Reply:
The whole `bottles` block handled by @BrewTestBot. I'm not sure but it looks like removing of it could add a merge conflict for `brew pull --bottle` process